core/templates/dev/head/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardBackendApiService.js

```diff
@@ -16,15 +16,27 @@
  * @fileoverview Service to retrieve information of topics and skills dashboard
```

Comment:
The file overview doesn't explain that this service can also perform PUT request.

Reply:
Done, changed it.